website/docs/plugin/framework/migrating/attributes-blocks/blocks-computed.mdx

```diff
@@ -0,0 +1,123 @@
+---
+page_title: 'Computed Blocks: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate computed blocks from SDKv2 to attribute validators in the plugin Framework.
+---
+
+# Computed Blocks
+
+Some providers, resources, and data sources include repeatable nested blocks in their attributes. Some blocks contain
+fields  with `Computed: true`, which means that the provider code can define the value or that it could come from the
+output of terraform apply (e.g., the ID of an EC2 instance). Refer to [Schemas - Blocks](/plugin/framework/FIXME) in the
+Framework documentation for details.
```

Comment:
```suggestion
output of terraform apply (e.g., the ID of an EC2 instance).
```

Per https://app.asana.com/0/1128567758154570/1202808060006727/f, we don't have docs for this yet.